src/components/resetPassword/ResetPasswordPage.js

```diff
@@ -0,0 +1,81 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { toast } from 'react-toastify';
+import PropTypes from 'prop-types';
+import ResetPasswordForm from './ResetPasswordForm';
+import { resetPassword } from '../../actions/userActions';
+
+export class ResetPasswordPage extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      password1: '',
+      password2: '',
+    };
+  }
+
+  componentDidMount() {
+    const { match: { params: { token } } } = this.props;
+    localStorage.setItem('auth_token', token);
```

Comment:
Since successful login has`auth_token`set in local storage, setting it here would mean that a user can access all the protected routes before the password is actually reset. Kindly consider changing `auth_token` to another variable name.

Reply:
This token is different from that of login so the user won't actually be able to access the other routes.
I will change the name though.

Reply:
@RonKbS, I tried it out and was able to access the dashboard.

Reply:
Update:
The dashboard is not protected.
Also, since I'm using the same Axios instance, I rely on the same key otherwise, as @MuhanguziDavid has experienced, the request is sent without a token if I change the key.
The token is removed after updating the password.

Reply:
@RonKbS, please consider using a different local storage key. It is not an `auth_token`. Its a `reset_token` or something more suitable. What you save it as in the front-end does not matter as long you name it to the right key before sending it to the backend.